lib/config/applyrule-targeted.cpp

```diff
@@ -88,15 +88,16 @@ bool ApplyRule::AddTargetedRule(const ApplyRule::Ptr& rule, const String& target
 }
 
 /**
- * If the given assign filter is like the following, extract the host names ("H", "h", ...) into the vector:
+ * If the given assign filter is like the following, extract the host names ("H", "h", ...) into the set:
  *
  * host.name == "H" [ || host.name == "h" ... ]
  *
- * The order of operands of || == doesn't matter.
+ * The order of operands of || == doesn't matter. All duplicates are also removed, e.g. not to initiate
+ * services twice when using a filter like `host.name == "h" || host.name == "h"`.
```

Comment:
Somewhat redundant: a set can't contain duplicates by definition, so that's not something explicitly done by this function.